projects/i18n/switch/language-switcher.loader.ts

```diff
@@ -0,0 +1,9 @@
+import {InjectionToken} from '@angular/core';
+
+import {TuiLanguageLoader} from './language-switcher.options';
+
+// @note: cannot be transferred to a shared file
+// ReferenceError: Cannot access 'TUI_LANGUAGE_LOADER' before initialization
+export const TUI_LANGUAGE_LOADER = new InjectionToken<TuiLanguageLoader>(
```

Comment:
Maybe we should make `tokens` folder in `i18n` like in the rest of projects and move all tokens there? 🤔 Just for consideration.

Reply:
fixed